apps/emqx/include/asserts.hrl

```diff
@@ -29,3 +29,17 @@
         )
     )
 ).
+
+-define(assertInclude(Pattern, List),
```

Comment:
Nit: it's nice to keep macro arguments distinct from variables by writing them in all caps.